### PR TITLE
test_force fails to create ios native apps

### PR DESCRIPTION
**Pull request by wmathurin**
test_force.js invokes forceios which internally does a pod install (which goes against the published pod specs).
Then test_force.js edit the pod file of the generated application to point to local copies of the Mobile SDK and then does a pod update.
That doesn't work now because the SalesforceAnalytics pod exists in development but not in production.

The solution is to have test_force.js disable the pod install call in forceios (actually it lives in createApp.sh) before executing forceios.

### CODE REVIEW

**Comment by wmathurin** [comment]
That change alone won't get the generation to succeed.
There are issues with the templates (its deployment target in iOS 8.0) and there seems to be issues with the pod spec as well.

File: node/test_force.js [hunk 1]
```diff
@@ -286,6 +287,16 @@ function editForceScriptToUseLocalPluginRepo(tmpDir, os) {
 }
 
 //
+// Update createApp.sh to not do pod install
+// 
+function editCreateAppToNotDoPodInstall(tmpDir) {
+    log('Editing  createApp.sh to not do pod install', COLOR.green);
+    miscUtils.replaceTextInFile(path.join(tmpDir, 'node_modules', 'forceios', 'build', 'app_template_files', 'createApp.sh'), new RegExp('pod install$', 'g'), '# pod install');
```
Comment (khawkins) [inline node/test_force.js]: Strictly speaking, wouldn't a more bullet-proof regexp be:

`new RegExp('^\s*pod install', 'g')`

?

**Comment by khawkins** [comment]
Otherwise, seems fine to me.

File: node/test_force.js (hunk 1 above)
Comment (wmathurin) [inline node/test_force.js]: Done here: https://github.com/forcedotcom/SalesforceMobileSDK-Shared/pull/408/commits/50e4233facf22fdc8baca49d1118611e7ddf6413

NB: 
it has to be either
new RegExp(/^\s*pod install/, 'g')
or
new RegExp('^\\s*pod install', 'g')

Comment (khawkins) [inline node/test_force.js]: Oh yeah, didn't pay attention to my escaping. 🙂  Update looks good to me!

**Pull request merged**